ui/core/src/model/time.ts

```diff
@@ -26,6 +26,14 @@ export interface RelativeTimeRange {
   pastDuration: DurationString;
 }
 
+export interface TimeOption {
+  from: string;
+  to: string;
+  display: string;
+}
```

Comment:
Suggestion: rather than having a new string format for this, why not use the existing `RelativeTimeRange` type from core? For example:

```ts
export interface TimeOption {
  value: RelativeTimeRange;
  display: string;
}
```

The new from/to string formats feel unstructured to me and it feels like there's no reason we need to do this like Grafana?

Reply:
The thought was keeping TIME_OPTIONS similar to how Grafana does it would make implementing shareable URLs easier since I got feedback from Charles that it'd be good to keep our URL structure the same as Grafana. But yeah, the approach currently isn't great since that needs to be customizable too so I'll change this

Reply:
Yeah, for URLs, we can always serialize/deserialize values to the query string however we want. Maybe those do end up looking more like what Grafana does. 🤷‍♂️